packages/cli/commands/sandbox/sync.js

```diff
@@ -39,9 +34,7 @@ exports.handler = async options => {
   const { force } = options; // For scripting purposes
   const accountId = getAccountId(options);
   const accountConfig = getAccountConfig(accountId);
-  const spinnies = new Spinnies({
-    succeedColor: 'white',
-  });
+  const env = getEnv(accountId) === 'qa' ? ENVIRONMENTS.QA : ENVIRONMENTS.PROD;
```

Comment:
I realize there are other commands doing this same thing, but I think we can make use of some existing utils here. Specifically [getValidEnv](https://github.com/HubSpot/hubspot-cli/blob/master/packages/cli-lib/lib/environment.js#L11).

You should be able to do `getValidEnv(getEnv(accountId))`